config/routes.rb

```diff
@@ -7,6 +8,9 @@
     get "/about", to: "static_pages#about"
     get "/contact", to: "static_pages#contact"
     post "/signup",  to: "users#create"
+    get    '/login',   to: 'sessions#new'
```

Comment:
- Dùng " thay cho '
- get "/login", to: "sessions#new" (cách 1 spaces thôi em, sửa tương tự cho các chỗ khác)